setup-env

```diff
@@ -128,10 +216,18 @@ END_OF_LINE
 fi
 
 # Create a new virtual environment for this project
-if ! pyenv virtualenv "${env_name}"; then
+#
+# If $PYTHON_VERSION is undefined then the system Python will be used.
```

Comment:
I believe this will actually use whatever is configured as the global version for `pyenv`. Would you please verify that and update accordingly?

Reply:
Yes, @mcdonnnj is correct here.

Reply:
I will get it updated now!

Reply:
See here: https://github.com/cisagov/skeleton-generic/pull/166/commits/01abde6f580cf549d77c81af9378d4a18812fa8f